build.sh

```diff
@@ -0,0 +1,3 @@
+(cd uems-event-micro-dionysus/; tsc)
```

Comment:
Is it safer to use npm commands here because locally installed typescript won't be in the path here because npm handles that

Reply:
I've always run these scripts assuming typescript is installed but I see your point - we can switch this to npm run build.

Reply:
Fixed